scene/gui/line_edit.cpp

```diff
@@ -1811,6 +1811,11 @@ bool LineEdit::has_selection() const {
 	return selection.enabled;
 }
 
+String LineEdit::get_selected_text() {
+	ERR_FAIL_COND_V(!selection.enabled, String());
```

Comment:
No selection is not an error, it probably should simply return empty string silently.

Reply:
Oh. My bad, I didn't think of this.